packages/kubernetes-1.15/kubernetes-1.15.spec

```diff
@@ -65,6 +64,7 @@ cp third_party/intemp/LICENSE LICENSE.intemp
 %build
 %cross_go_configure %{goimport}
 export KUBE_BUILD_PLATFORMS="linux/%{_cross_go_arch}"
+export GOLDFLAGS="-buildid="
 make WHAT="cmd/kubelet"
```

Comment:
My preferred way to accomplish this would be:
```
make WHAT="cmd/kubelet" GOLDFLAGS="-buildmode=pie"
```

That would also let us drop the "enable PIE for platform binaries" patch, and it would avoid the concern that we might no longer be randomly generating a build ID for binaries, which is required to distinguish debug symbols for different builds. We definitely don't want all copies of `kubelet` to have the same build ID.

I also think this would be cleaner as a single patch rather than three patches with the same commit message.